common/test/acceptance/pages/studio/course_info.py

```diff
@@ -1,16 +1,134 @@
 """
 Course Updates page.
 """
-
+from common.test.acceptance.pages.common.utils import click_css, confirm_prompt
 from common.test.acceptance.pages.studio.course_page import CoursePage
+from common.test.acceptance.pages.studio.utils import type_in_codemirror, set_input_value
 
 
 class CourseUpdatesPage(CoursePage):
     """
     Course Updates page.
     """
-
     url_path = "course_info"
 
     def is_browser_on_page(self):
-        return self.q(css='body.view-updates').present
+        """
+        Returns whether or not the browser on the page and has loaded the required content
+        """
+        # Check for the presence of handouts-content, when it is present the render function has completed
+        # loading the updates and handout sections
+        return self.q(css='.handouts-content').present and \
```

Comment:
Nit (which I don't personally care about): we avoid \ per PEP-8 guidelines.

https://www.python.org/dev/peps/pep-0008/

The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.